src/lib/crypto/kimchi_bindings/wasm/src/arkworks/bigint_256.rs

```diff
@@ -29,15 +29,18 @@ impl FromWasmAbi for WasmBigInteger256 {
     type Abi = <Vec<u8> as FromWasmAbi>::Abi;
     unsafe fn from_abi(js: Self::Abi) -> Self {
         let bytes: Vec<u8> = FromWasmAbi::from_abi(js);
-        WasmBigInteger256(BigInteger256(FromBytes::read(bytes.as_slice()).unwrap()))
+        // TODO this used FromBytes before arkworks 0.4.2, check serialization is consistent after update
```

Comment:
cc @volhovm for this TODO. I guess it is correct if we do have regression tests?

Reply:
Ignore. It is 3.0.2 into master. Not 3.0.3